### ISSUE TITLE
Form Settings Tab is less than accesible 

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/Caldera-Forms#936]
Missing for attributes on labels and other accessibility markups

Placing on 1.4.5 and doing now so I can modify that tab in 1.5

**Comment by Shelob9** [comment]
Close via 	ac8b47e #mantelope
